python/cudf/dataframe/series.py

```diff
@@ -1224,6 +1224,15 @@ def mean_var(self, ddof=1):
     def sum_of_squares(self, dtype=None):
         return self._column.sum_of_squares(dtype=dtype)
 
+    def round(self, decimals=0):
```

Comment:
We should move this down to the `Column` level instead of at the Series level and then have the Series level function call that instead.

Reply:
Moved the core functionality down to the Column level.